### ISSUE TITLE
Starting webpack from directory without configuration should display a warning

### ISSUE THREAD

**Issue by ipavlic** [webpack/webpack#2260]
When webpack is started from a folder that doesn't have a configuration file, it outputs `Output filename not configured.`

It should also say that no configuration is used to avoid confusion.

**Comment by bebraw** [comment]
This has been fixed in webpack 2:

```bash
No configuration file found and no output filename configured via CLI option.
A configuration file could be named 'webpack.config.js' in the current directory.
Use --help to display the CLI options.
```

**Comment by Roshan1729** [comment]
Hi @bebraw  , I am using the webpack version 2.4.1 and i am still getting the same error . I even though created the configuration file named 'webpack.config.js' in the current directl. Any suggestion why would that be happening and what could be done about it?

**Comment by bebraw** [comment]
@Roshan1729 Can you open a separate issue with more info?

**Comment by ramintagizade** [comment]
Should we have a folder named node_modules on every project developed with React JS then ?

**Comment by bebraw** [comment]
@ramintagizade Can you open a separate issue with more info?

**Comment by shashikantChetu** [comment]
No configuration file found and no entry configured via CLI option.
When using the CLI you need to provide at least two arguments: entry and output.
A configuration file could be named 'webpack.config.js' in the current directory.
Use --help to display the CLI options.

**Comment by amdcaruso** [comment]
Facing the same issue.

**Comment by cschaepe** [comment]
I have been struggling with this as well webpack version 3.6.0.  It seems to want to run it perhaps where it was installed globally not at the project level.  The output seems to indicate the correct project sub directory but it only works by moving the webpack.config.js file up to the directory where webpack was installed globally.

**Comment by bhargodevarya** [comment]
I created a project to try my hands at npm, webpack, babel. I have webpack installed globally.
In my project after installing webpack as dev dependency and creating a webpack.config.js file and creating a target in the package.json file , build, which invokes webpack as following:
"build": "webpack"
I get an error stating the the config file is not present.
I tried to uninstall webpack globally by npm uninstall webpack -g. This also did not resolve the issue.
Therefore I used th complete path of webpack.js from inside the node_modules folder and gave the --config flag to set the value of my webpack config file.
Now this works!!!
